cli/delete.go

```diff
@@ -15,9 +15,17 @@ import (
 var confirmed bool
 
 type deleteOptions struct {
-	Revert bool
+	Revert         bool
+	ExcludeRecords bool
 }
 
+const (
+	deleteProjectConfirmation = "Deleting project...Please confirm [y/N]: "
+	deleteRecordConfirmation  = "Deleting record...Please confirm [y/N]: "
+	deleteRecordsWarning      = "Do you wish to delete project records? Please confirm [y/N]: "
+	revertRecordsWarning      = "Do you wish to restore project records from backups?\nWarning! This will overwrite any changes made after the most recent backup. Please confirm [y/N]: "
```

Comment:
We could use a multiline string for this:

```suggestion
	revertRecordsWarning      = `Do you wish to restore project records from backups?
Warning! This will overwrite any changes made after the most recent backup. Please confirm [y/N]: `
```